README.md

```diff
@@ -1,6 +1,6 @@
 # TEAM SCORCHIO: MAKERS BnB
 ----------
-
+THIS IS A CHANGE IN THE README
 ## HOW TO USE
```

Comment:
What a silly thing to add to the readme!!!!!

Reply:
:P